### PR TITLE
Fixing positional resource id descriptions

### PR DESCRIPTION
**Pull request by zikalino**
I have fixed some other places and replaced **<id>** with **<resource-id>** in command descriptions as it's more common and descriptive pattern.

### CODE REVIEW

File: commands/registry.go [hunk 1]
```diff
@@ -80,12 +80,12 @@ func Registry() *Command {
 	cmdRunRegistryCreate.Example = `The following example creates a registry named ` + "`" + `example-registry` + "`" + ` in the NYC3 region: doctl registry create example-registry --region=nyc3`
 
 	getRegDesc := "Retrieves details about a private container registry, including its name and the endpoint used to access it."
-	cmdRegistryGet := CmdBuilder(cmd, RunRegistryGet, "get", "Retrieve details about a container registry",
+	cmdRegistryGet := CmdBuilder(cmd, RunRegistryGet, "get <registry-id>", "Retrieve details about a container registry",
```
Comment (andrewsomething) [inline commands/registry.go:83]: At the moment, only one container registry is supported per account. So this command is a bit of an outlier. There is no positional argument, just `doctl registry get`.

File: commands/registry.go [hunk 2]
```diff
@@ -80,12 +80,12 @@ func Registry() *Command {
...
 		getRegDesc, Writer, aliasOpt("g"), displayerType(&displayers.Registry{}))
 	cmdRegistryGet.Example = `The following example retrieves details about a registry named ` + "`" + `example-registry` + "`" + `: doctl registry get example-registry`
 
 	deleteRegDesc := "Permanently deletes a private container registry and all of its contents. This is irreversible."
-	cmdRunRegistryDelete := CmdBuilder(cmd, RunRegistryDelete, "delete",
+	cmdRunRegistryDelete := CmdBuilder(cmd, RunRegistryDelete, "delete <registry-id>",
```
Comment (andrewsomething) [inline commands/registry.go:88]: Same thing here.

**Review by andrewsomething — CHANGES_REQUESTED** [review]
Thanks for another great contribution! I left a couple comments inline regarding the container registry commands. They do not actually require a positional argument.

File: commands/registry.go (hunk 1 above)
Comment (zikalino) [reply commands/registry.go:83]: oops.. you're right..
it actually works without registry id. i was specifying registry id as first parameter but seems it was actually ignored.....

**Comment by zikalino** [comment]
ok, removed registry changes. i left other improvements - probably still worth merging.

**Review by andrewsomething — APPROVED** [review]

**Comment by andrewsomething** [comment]
Definetly! Thanks again!

**Pull request merged**